integration-tests/src/common/java/pl/allegro/tech/hermes/integrationtests/setup/HermesExtension.java

```diff
@@ -128,7 +129,11 @@ private void removeGroups() {
         GroupService service = management.groupService();
         List<Group> groups = service.listGroups();
         for (Group group : groups) {
-            service.removeGroup(group.getGroupName(), testUser);
+            try {
+                service.removeGroup(group.getGroupName(), testUser);
+            } catch (GroupNotEmptyException ignored) {
+                // ignore
```

Comment:
Sometimes HierarchicalCache causes problems when deleting groups, I decided to ignore these errors, which in the worst case can end up with some group not being deleted which does not affect the tests

Reply:
+1, minor: please consider adding logging here as it can help in debbuging/analyzing tests in the future